test/virt/quad/topology.dot.in

```diff
@@ -24,6 +24,7 @@ graph "quad" {
 	    label="{ <e1> e1 | <e2> e2 | <e3> e3 | <e4> e4 } | dut1 | { <e5> e5 | <e6> e6 | <e7> e7 | <e8> e8}",
 	    pos="10,30!",
 	    provides="infix",
+	    boot="primary",
```

Comment:
`boot="<part>"` sounds like _configuration_ to me, i.e., if I change this to `"secondary"` and run `qeneth restart`, then the device will boot on that partition instead.

Maybe `expected_boot="primary"`?